Clearly tell user that E2EE has been enabled for an account

claucambra opened the pull request:
![Screenshot 2022-11-15 at 12 26 51](https://user-images.githubusercontent.com/70155116/201908558-13c06578-27a1-473a-810a-051d9cdd0214.png)

![Screenshot 2022-11-15 at 12 26 54](https://user-images.githubusercontent.com/70155116/201908606-8fc0c8f8-4985-44e2-acfe-93e8806bcc03.png)




Part of https://github.com/nextcloud/desktop/issues/5126

Signed-off-by: Claudio Cambra <claudio.cambra@nextcloud.com>

<!-- 
Thanks for opening a pull request on the Nextcloud desktop client.

Instead of a Contributor License Agreement (CLA) we use a Developer Certificate of Origin (DCO).
https://en.wikipedia.org/wiki/Developer_Certificate_of_Origin

To accept that DCO, please make sure that you add a line like
Signed-off-by: Random Developer <random@developer.example.org>
at the end of each commit message.

This Signed-off-by trailer can be added automatically by git if you pass --signoff or -s to git commit.
See also:
https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---no-signoff
-->

jancborchardt reviewed (changes requested):
Nice! :) Just the icons are a bit off:
- The checkmark icon shouldnt be the "All synced" icon
- The info icon looks a bit oldish

Similar like for "General" and "Network", could we use native icons for each of them which work for the platform?

Otherwise as quick fixes:
- E2E active, a lock: https://fonts.google.com/icons?selected=Material%20Icons%3Alock%3A
- E2E possible, info i: https://fonts.google.com/icons?selected=Material%20Icons%3Ainfo%3A

claucambra commented on src/gui/accountsettings.cpp:
```diff
@@ -246,9 +246,12 @@ void AccountSettings::slotE2eEncryptionMnemonicReady()
-    _ui->encryptionMessage->setText(tr("This account supports End-to-End encryption"));
+
+    _ui->encryptionMessage->setMessageType(KMessageWidget::Positive);
+    _ui->encryptionMessage->setText(tr("End-to-End encryption has been enabled for this account"));
```
Yes, I think so, as the server supports several types of encryption systems and "encryption" is not clear which one

claucambra replied:
Then again, if it is a contextual string (e.g. a messagebox message) that should be clear enough

jancborchardt replied:
Btw, the wording or capitalization should be End-to-end encryption, with lowercase second e. And if in a sentence, the first e should be lowercase as well of course.

claucambra replied:
Thanks, this is now done :)

claucambra commented:
> Nice! :) Just the icons are a bit off:
> 
> * The checkmark icon shouldnt be the "All synced" icon
> * The info icon looks a bit oldish
> 
> Similar like for "General" and "Network", could we use native icons for each of them which work for the platform?
> 
> Otherwise as quick fixes:
> 
> * E2E active, a lock: https://fonts.google.com/icons?selected=Material%20Icons%3Alock%3A
> * E2E possible, info i: https://fonts.google.com/icons?selected=Material%20Icons%3Ainfo%3A

Okay, I have changed the icons used in the message box and updated the screenshots above 🙂

Ad for native icons more generally, let's open a separate issue for this

claucambra commented on src/gui/accountsettings.cpp:
```diff
@@ -246,9 +246,12 @@ void AccountSettings::slotE2eEncryptionMnemonicReady()
+    _ui->encryptionMessage->setIcon(Theme::createIconFromSvgResource(QStringLiteral(":/client/theme/colored/state-ok.svg")));
     _ui->encryptionMessage->show();
-    
+
```
Removed

claucambra replied:
removed

jancborchardt reviewed (approved):
Noice! :)

tobiasKaminsky commented:
Not sure if it is still valid, but in second screenshot:
I assume that account has no E2E yet, so button might better be "setup" instead of "enable e2e" to distinguish this more.

claucambra commented:
> Not sure if it is still valid, but in second screenshot:
> I assume that account has no E2E yet, so button might better be "setup" instead of "enable e2e" to distinguish this more.

changed :)
